go.mod

```diff
@@ -431,3 +431,5 @@ replace (
 
 // Exclude this version because the version has an invalid checksum.
 exclude github.com/docker/distribution v2.8.0+incompatible
+
+replace github.com/elastic/elastic-agent-libs => github.com/belimawr/elastic-agent-libs v0.2.9-0.20240423125757-a73eab058a9f
```

Comment:
Just a heads up: the replace is still here after all the changes and the PR cannot be merged yet.

Reply:
Thanks Denis! I'm well aware of this, I got a few comments on https://github.com/elastic/elastic-agent-libs/pull/171, hopefully it will be merged soon.

Reply:
Done 89db14b!